go.mod

```diff
@@ -61,3 +60,6 @@ replace (
 	google.golang.org/genproto => google.golang.org/genproto v0.0.0-20191002211648-c459b9ce5143
 	google.golang.org/grpc => google.golang.org/grpc v1.24.0
 )
+
+// WORKAROUND until k8s v1.18+ is not present in knative/pkg
```

Comment:
Just curious to understand this a bit more. do you mean is present, or is there a particular issue with 1.18?

Reply:
haha, sorry for the confusing wording :) I mean is present (the client api was changed a little bit in k8s 1.18), knative should move to k8s in a near future. I am gonna rephrase the comment